bands/templates/bands/agreement-base.html

```diff
@@ -6,7 +6,7 @@
     Band agreement for <strong>{{ band.group.name }}</strong> to perform at {{ c.EVENT_NAME_AND_YEAR }}!
     </h2>
     <p>
-        {{ c.EVENT_NAME_AND_YEAR }} will be held on <strong>{{ c.EPOCH|datetime_local("%B %d") }} - {{ c.ESCHATON|datetime_local("%d, %Y") }}.</strong><br>
+        {{ c.EVENT_NAME_AND_YEAR }} will be held on <strong>{{ c.EPOCH|datetime_local("%B %-d") }} - {{ c.ESCHATON|datetime_local("%-d, %Y") }}.</strong><br>
```

Comment:
The dash in the date formatting string `%-d` removes zero padding, so we get `January 4 - 7` instead of `January 04 - 07`.

Reply:
Oh neat, good to know!